### ISSUE TITLE
Add structured data

### ISSUE THREAD

**Issue by chrfritsch** [molebox/gatsby-theme-seo#3]
It would be nice if the SEO component would also output structured data like https://github.com/LeKoArts/gatsby-starter-prismic/blob/master/src/components/SEO/SEO.jsx

**Comment by molebox** [comment]
Yeah that looks really nice. Im also planning on ditching the static query component in favor of the useStaticQuery hook. 

Im kinda knees deep in another theme at this moment but can get on this at a later date. If you feel like adding it now you're welcome to submit a PR otherwise. :)
